tests/component/sections/shared/add-data-actions/AddDataActionsButton.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import AddDataActionsButton from '../../../../../src/sections/shared/add-data-actions/AddDataActionsButton'
+
+describe('AddDataActionsButton', () => {
+  it('renders the button', () => {
+    cy.customMount(<AddDataActionsButton />)
+
+    cy.findByRole('button', { name: /Add Data/i }).should('exist')
+  })
+
+  // it('renders the tooltip', () => {  })
```

Comment:
I don't see the tooltip in the JSF version, is that something that we want to add to the spa?